### PR TITLE
CLI mnemonic seed phrase

### PR DESCRIPTION
**Pull request by matibat**
Implements generateSeedPhrase() and generateJWKWallet(). Also adds the commands to commander for it to run

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -173,6 +173,40 @@ program
...
+program.command('generate-seedphrase').action(async () => {
+	const arweave = Arweave.init({
+		host: 'arweave.net', // Arweave Gateway
+		//host: 'arweave.dev', // Arweave Dev Gateway
+		port: 443,
+		protocol: 'https',
+		timeout: 600000
+	});
+	const walletDao = new WalletDAO(arweave);
+	const seedPhrase = walletDao.generateSeedPhrase();
```
Comment (fedellen) [inline src/index.ts:185]: Need an `await` here

File: src/wallet_new.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import * as B64js from 'base64-js';
 import * as crypto from 'crypto';
 import jwkToPem, { JWK } from 'jwk-to-pem';
 import Arweave from 'arweave';
+import mnemonicKeys from 'arweave-mnemonic-keys';
```
Comment (fedellen) [inline src/wallet_new.ts:6]: This results in mnemonicKeys being `undefined` for me, this fixes it:
```suggestion
import * as mnemonicKeys from 'arweave-mnemonic-keys';
```

File: src/index.ts (hunk 1 above)
Comment (matibat) [reply src/index.ts]: Fixed here: 9bbcb0033d77d0a1263fbe11cefa41fcc6013222

File: src/wallet_new.ts (hunk 2 above)
Comment (matibat) [reply src/wallet_new.ts]: Fixed here: 9bbcb0033d77d0a1263fbe11cefa41fcc6013222

**Review by fedellen — APPROVED** [review]

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**